api/buffer/design.go

```diff
@@ -477,7 +477,10 @@ var Mapping = Type("Mapping", func() {
 
 var Column = Type("Column", func() {
 	Description("An output mapping defined by a template.")
-	Attribute("primaryKey", Boolean, "Sets this column as a part of the primary key of the destination table.")
+	Attribute("primaryKey", Boolean, func() {
+		Description("Sets this column as a part of the primary key of the destination table.")
+		Default(false)
+	})
```

Comment:
Bez default hodnoty sa tu zbytocne generuje `*bool` + nie je to v responses (ked to nema default + nie je to required).